### ISSUE TITLE
Crash happened when using some English words for full text search

### ISSUE THREAD

**Issue by guoyingtao** [Tencent/wcdb#367]
### The language of WCDB

Swift

### The version of WCDB

1.0.7.3

### The platform of WCDB

iOS

### The installation of WCDB

Cocoapods

### What's the issue?

![item-0_and_displaysextra_and_item-0_and_item-0_and_item-2_and_item-1_and_item-0_and_item-0](https://user-images.githubusercontent.com/26723384/44800747-26843a00-ab85-11e8-8a2f-be0452044abf.jpg)

> Explain what you want by example or code **in English**.

When I used some English words like "a", "go", "goo", "tod" for full-text search, the crash happened.
It seems a bug from Swift.

I tried enumerateLinguisticTags with the same English words in a Playground and found it crashed too.

**Comment by guoyingtao** [comment]
Why was this issue added wontfix label? You can not use full-text English search feature if not fixing it.

**Comment by RingoD** [comment]
It's a bug inside the Swift builtin framework which we already report to Swift staff at [SR-7330](https://bugs.swift.org/browse/SR-7330).

As the release note said, ones should use Swift 4.2(Xcode 10) to avoid this crash.

**Comment by guoyingtao** [comment]
I tested it with Swift 4.2 (Xcode10.0 Beta4) and found it still crashed.

**Comment by RingoD** [comment]
It's fine on my Xcode. If it's still unstable, I think you should report more issue to Swift staffs to push them fixing.

**Comment by guoyingtao** [comment]
I just downloaded Xcode10.0 beta6 and tested it again. It crashed again.
If you use some words like "today", it is OK. but if you use some word like "a", "aa", "tod", it will crash.

And even I report the issue to Swift staffs, what if it takes a long time to fix it? Then what users can do is just waiting and not using English full-text search with WCDB?

**Comment by guoyingtao** [comment]
In my opinion, if it still crashes because of this reason, maybe a temporary fix would be a better solution. Then we can also wait for the fix from Swift staffs. At least users can use WCDB without worrying about this kind of crash before the Swift fix.

**Comment by RingoD** [comment]
In fact, not only `String` but also a lot of classes/methods are unstable, which we already reported to Swift staffs. 
Fixing this only make no sense. 
We do not tend to, and inability to handle all issues of Swift.

**Comment by guoyingtao** [comment]
I don't think it makes no sense. 
I am making an app with Swift and I want to do some full-text search with Chinese + English. I tried a couple of SQLite tokenizers but no one worked very well. Finally, I found WCDB.swift, it can do a perfect Chinese full-text search. But when I do some search with Chinese + English, my app crashes.
I don't think I am the only user who is using WCDB.swift like this.
I think a working library is better than a crashing one although maybe its code looks perfect.
